ApollosProject/apollos-apps
# fix: Update Prayer Experience Primary navigation

## ✏️ Description
On the current implementation of the Prayer Swiper Experience, there is an issue with the primary navigation on Android in which the buttons are not tappable. In order to solve this, I've increased the `zIndex` of the primary navigation from `100` to `1000` which seems to have solved the issue

## 🔬 To Test
1. Open up the app on an Android device
2. Navigate to the Daily Prayer experience
3. While on the main Swiper, interact with each of the 3 primary navigation elements

**Make sure you set the `--variant=release` flag on the `android` command in scripts as this error seems to be present only in production**

## 📸 Screenshots
Borked

https://user-images.githubusercontent.com/45076058/211418876-7f28f3f7-7c60-41de-9d5e-c26a0d06f745.mp4

Fixed

https://user-images.githubusercontent.com/45076058/211418900-380a9da1-3ad5-4f2e-9a7d-07a57f28b13e.mp4


## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
